packages/terra-alert/src/terra-dev-site/test/alert/Alert.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from 'terra-button';
+import Alert from '../../../Alert';
+
+class AlertActionButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      actionButtonClickCount: 0,
+      showAlert: false,
+    };
+    this.actionFunc = this.actionFunc.bind(this);
+    this.popAlert = this.popAlert.bind(this);
+  }
+
+  actionFunc() {
+    this.setState(prevState => ({ actionButtonClickCount: prevState.actionButtonClickCount + 1 }));
+  }
```

Comment:
I don't think this is an actual use-case of how we intent alert actions to be used. I would expect the action to dismiss the the alert, not update the text. Since is added to show how alerts are being handled with VoiceOver, can we update this to close the alert?

Reply:
Changed  33871d2